hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -40,6 +40,7 @@
     note_deprecation,
     settings as Settings,
 )
+from hypothesis import PrintSettings
```

Comment:
Apparently our formatters want this line moved about the imports from `hypothesis._settings`... to be honest I never remember how this should go, but `pip install isort` then `isort --apply --recursive .` will fix them for you.

(`pip install black` then `black .` is our other formatter, but it's happy with this pull request)

Reply:
Fixed.